core/templates/dev/head/base_components/BaseContentDirective.ts

```diff
@@ -17,9 +17,8 @@
  */
 
 require('pages/OppiaFooterDirective.ts');
-
```

Comment:
Undo this change. (There should be one newline in between the directive and service imports ([ref.](https://github.com/oppia/oppia/wiki/Coding-style-guide#adding-require-to-the-typescript-files-with-directive)))

Reply:
Done!

Reply:
@Showtim3, Sorry I think my first comment wasn't clear, I wanted to make it:
```
require('directive')

require('services')
require('services')
```

i.e, directive and service imports should be separated by an empty line.

Reply:
@oppia/dev-workflow-team, It would be great if linter can catch this. (feature-request)

Reply:
HI @DubeySandeep, we have #6748 to keep its track.